includes/admin/class-wc-admin-addons.php

```diff
@@ -794,4 +794,20 @@ public static function show_extension( $item ) {
 
 		return true;
 	}
+
+	/**
+	 * We're displaying page=wc-addons and page=wc-addons&section=helper as two separate pages.
+	 * When we're on those pages, add body classes to distinguishe them.
+	 *
+	 * @param string $admin_body_class Unfiltered body class.
+	 *
+	 * @return string Body class with added class for Marketplace or My Subscriptions page.
+	 */
+	public static function filter_admin_body_classes( string $admin_body_class = '' ): string {
+		if ( isset( $_GET['section'] ) && 'helper' === $_GET['section'] ) {
+			return " $admin_body_class woocommerce-page-wc-subscriptions ";
+		}
+
+		return " $admin_body_class woocommerce-page-wc-marketplace ";
+	}
```

Comment:
I'm not seeing the classes for WooCommerce > Settings or WooCommerce > Status pages registered in a similar way, but I can see there's a CSS class added for each of them (`woocommerce_page_wc-settings` and `woocommerce_page_wc-status` respectively). 

This is not a blocker but in the long run it would be great to find out how exactly these classes are added for the Settings or Status pages and add it for Marketplace and My Subscriptions in a similar way. 

Also we can keep the convention of `woocommerce_page_wc-{pagename}` here as well.

Reply:
I originally had the new classes like `woocommerce_page_wc-{pagename}`, but stylelint has the `selector-class-pattern` rule, which demands kebab case, so I changed them to that.

It took me ages to find the source of the original classes, and to discover there's no way to filter them. They are set in WordPress core, in `wp-admin/admin-header.php:81`:

```php
$admin_body_class = preg_replace( '/[^a-z0-9_-]+/i', '-', $hook_suffix );
```

and 227-8, where they bypass the filter:

```php
$admin_body_classes = apply_filters( 'admin_body_class', '' );
$admin_body_classes = ltrim( $admin_body_classes . ' ' . $admin_body_class );
```

Reply:
Thanks for checking and explaining this, we should be good to merge in that case 🙂